### PR TITLE
dotter: Add version 0.12.6

### PR DESCRIPTION
**Pull request by rashil2000**

### CODE REVIEW

**Comment by issaclin32** [comment]
Please be kind.
At least add some brief information about **"what the application does"** in your PR,
rather than *No description*, or throwing a link and let the maintainers to figure out.

**Comment by rashil2000** [comment]
Hey @issaclin32, I'm extremely sorry. I don't usually open blank issues/PRs. I had typed out the description from dotter's Readme; no idea what happened to the GitHub page when I clicked the `Comment` button. Would like to apologize for that.

Here's the description:
> Dotter is a dotfile manager and templater. It provides a flexible configuration and automatic templating or symlinking to the target locations.

**Comment by issaclin32** [comment]
> Hey @issaclin32, I'm extremely sorry. ...

No worries, just want to remind you.

File: bucket/dotter.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.12.6",
+    "description": "A dotfile manager and templater written in rust",
```
Comment (issaclin32) [inline bucket/dotter.json:3]:
```suggestion
    "description": "Dotfile manager and templater",
```

File: bucket/dotter.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.12.6",
+    "description": "Dotfile manager and templater",
+    "homepage": "https://github.com/SuperCuber/dotter",
+    "license": "Unlicense",
```
Comment (issaclin32) [inline bucket/dotter.json:5]:
```suggestion
    "license": "Unlicense",
    "notes": "Run 'dotter init' to initialize config file",
```

Comment (rashil2000) [reply bucket/dotter.json:5]: I'm not sure if this line should be added. The `dotter init` command should only be run inside a particular directory (where you want to save/load your configs from). As mentioned in the wiki, this should be a local git repository.

image

**Review by issaclin32 — COMMENTED** [review]
How about adding this instead?
`"notes": "To set up dotter, check instructions on 'https://github.com/SuperCuber/dotter/wiki/Setup-and-Configuration'`

**Comment by rashil2000** [comment]
Done!

**Review by issaclin32 — DISMISSED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**